src/leiningen/new/figwheel/index.html

```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <link href="css/style.css" rel="stylesheet" type="text/css">{{#reagent?}}
-    <script src="http://fb.me/react-0.12.1.js"></script>{{/reagent?}}
```

Comment:
Not necessary with new reagent https://github.com/reagent-project/reagent/blob/master/project.clj#L8